terraform/ecs_ec2/daemon/main.tf

```diff
@@ -225,6 +226,7 @@ resource "aws_ecs_service" "extra_apps_service" {
   task_definition = aws_ecs_task_definition.extra_apps_task_definition.arn
   desired_count   = 1
   launch_type     = "FARGATE"
+  enable_execute_command = true
```

Comment:
Added `enable_execute_command` for debug purpose in case we ever need to run command  on the container itself.